UnityProject/Assets/Scripts/Items/Others/RandomItemSpot.cs

```diff
@@ -107,7 +107,7 @@ private void GenerateItem(PoolData poolData, bool spawn)
 			}
 
 			var item = poolData.RandomItemPool.Pool.PickRandom();
-			var spread = fanOut ? Random.Range(-0.5f, 0.5f) : (float?) null;
+			var spread = fanOut ? Random.Range(0, 0.5f) : (float?) null;
```

Comment:
why?

Reply:
- is reserved for custom behaviour

Reply:
and due to how the code works 0.5f makes it a random direction in any direction

Reply:
since it goes random(spread, -spread )